### DISCUSSION TITLE
Slides are visible outside swiper container

### DISCUSSION THREAD

**Discussion by kdoan6789** [nolimits4web/swiper#4360] [Q&A] [answered]
Hi everyone, 

I got a problem while using swiper js for my project. Slides that are outside swiper container are still visible isntead of hidden. Please have a look at the attachment to see better. I have tried to find solution for few days but couldn't fix it. And this problem only occur with 2 out of 3 carousels I used. Any helps are appreciated. 

HTML: 
<div class="swiper-container test-container">
    <h4>Container</h4>
    <div class="swiper-wrapper">
      <div class="swiper-slide">test1</div>
      <div class="swiper-slide">test2</div>
      <div class="swiper-slide">test3</div>
      <div class="swiper-slide">test4</div>
    </div>
    <div class="nav">
      <div class="swiper-button-next"></div>
      <div class="swiper-button-prev"></div>
    </div>
</div>

JS:
var testSwiper = new Swiper('.test-container', {
  nextButton: '.test-container .swiper-button-next', 
  prevButton: '.test-container .swiper-button-prev',
  pagination: '.test-container .swiper-pagination',
  grabCursor: true,
  autoplay: false,
  slidesPerView: 3,
  spaceBetween: 10,
  centeredSlides: true, 
  loop: true
});

<img width="1881" alt="Screen Shot 2021-03-21 at 10 55 09 PM" src="https://user-images.githubusercontent.com/58367067/111911531-91295800-8a98-11eb-8982-67847cbdef1e.png">

**Comment by nolimits4web** [comment, marked as answer]
For me looks like you did something with styles. Make sure there is a overflow hidden set on swiper-container

**Reply by kdoan6789** [reply]
You're right, adding overflow hidden fixed the problem. It's just weird that I hadn't done anything with styles for container. Thank you very much.
